hero_trump/card.py

```diff
@@ -53,27 +59,90 @@ def card_info(self):
               "Tools:", self.tools,
               )
 
-c1 = Card("null", 0, "null", 0, 0, 0, 0, 0, 0, False)
-c2 = Card("null", 1, "null", 0, 0, 0, 0, 0, 0, False)
-c3 = Card("null", 2, "null", 0, 0, 0, 0, 0, 0, False)
-c4 = Card("null", 3, "null", 0, 0, 0, 0, 0, 0, False)
-c5 = Card("null", 4, "null", 0, 0, 0, 0, 0, 0, False)
-c6 = Card("null", 5, "null", 0, 0, 0, 0, 0, 0, False)
-c7 = Card("null", 6, "null", 0, 0, 0, 0, 0, 0, False)
-c8 = Card("null", 7, "null", 0, 0, 0, 0, 0, 0, False)
-c9 = Card("null", 8, "null", 0, 0, 0, 0, 0, 0, False)
-c10 = Card("null", 9, "null", 0, 0, 0, 0, 0, 0, False)
-
-c1.assign_card()
-c2.assign_card()
-c3.assign_card()
-c4.assign_card()
-c5.assign_card()
-c6.assign_card()
-c7.assign_card()
-c8.assign_card()
-c9.assign_card()
-c10.assign_card()
-
-c4.card_info()
+class Deck:
+    def __init__(
+        self,
+        cards,
+    ):
+        self.cards = cards
+        self.length = len(cards)
+
+    def read_cards(self):
+        with open('Cards.csv', 'r') as file:  
+            reader = csv.reader(file)
+            for row in reader:
+                newcard = Card()
+                newcard.assign_card(row)
```

Comment:
with my initi, suggestion, the row arg would go in the line above and this line would no longer be needed.